GitUI/CommandsDialogs/FormRebase.Designer.cs

```diff
@@ -193,6 +193,7 @@ private void InitializeComponent()
             // 
             this.Currentbranch.Anchor = System.Windows.Forms.AnchorStyles.Left;
             this.Currentbranch.AutoSize = true;
+            this.Currentbranch.Font = new System.Drawing.Font("Segoe UI", 9F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point);
```

Comment:
Are we doing this in other dialogs? Would it look correct when the user configures a different font?

Reply:
You're right. I did it another way that will respect the user selected font...